src/includes/set-level/android.mdx

```diff
@@ -0,0 +1,3 @@
+```java
+Sentry.setLevel(SentryLevel.WARNING);
```

Comment:
unclear what this would even do fwiw.. smells wrong to me

Reply:
basically, it forwards to the current `scope.setLevel(...)`, this was made to keep compatibility/easy migration with older Java/Android versions 1.x

Reply:
is it Sentry or SentryAndroid?

Reply:
nvm saw other comment